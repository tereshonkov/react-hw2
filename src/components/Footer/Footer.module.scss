.wrapper {
    display: flex;
    justify-content: center;
    align-items: center;
    background: rgb(60, 66, 66);
    padding: 60px 110px;
}
.text {
    color: rgb(255, 255, 255);
    font-size: 18px;
    font-weight: 700;
}