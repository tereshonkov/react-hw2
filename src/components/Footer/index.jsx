import React from 'react'
import styles from './Footer.module.scss'

export default function index() {
  return (
    <div className={styles.wrapper}>
        <p className={styles.text}>Copyright Â© 2023 Euphoria Folks Pvt Ltd. All rights reserved.</p>
    </div>
  )
}
