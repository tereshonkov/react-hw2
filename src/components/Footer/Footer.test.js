import Footer from "./index";
import { render, screen } from "@testing-library/react";

describe("Footer text render", () => {
  test("should render the copyright text", () => {
    const { asFragment } = render(<Footer />);

    const paragraph = screen.getByText(
      "Copyright Â© 2023 Euphoria Folks Pvt Ltd. All rights reserved."
    );
    expect(paragraph).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
