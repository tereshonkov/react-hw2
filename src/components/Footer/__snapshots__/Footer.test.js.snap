// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer text render should render the copyright text 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    <p
      class="text"
    >
      Copyright Â© 2023 Euphoria Folks Pvt Ltd. All rights reserved.
    </p>
  </div>
</DocumentFragment>
`;
